Rakefile

```diff
@@ -30,3 +30,99 @@ task :docstats, [:path] do |_, args|
 end
 
 Rake::ExtensionTask.new('drawText')
+
+desc 'Create a new version of the release-toolkit gem'
+task :new_release do
+  version_file = File.join('lib', 'fastlane', 'plugin', 'wpmreleasetoolkit', 'version.rb')
+  require_relative(version_file)
+  puts ">>> Current version is: #{Fastlane::Wpmreleasetoolkit::VERSION}"
+  puts ">>> Pending CHANGELOG:\n" + get_changelog_section(file: 'CHANGELOG.md', section_title: 'Develop').map { |l| "| #{l}" }.join
+
+  puts '>>> New version to use?'
+  new_version = STDIN.gets.chomp
+
+  ### VERSION constant
+  puts '>>> Updating `VERSION` constant in `version.rb`...'
+  content = File.read(version_file)
+  content.gsub!(/VERSION = .*/, "VERSION = '#{new_version}'")
+  File.write(version_file, content)
+  sh('bundle', 'install') # To update Gemfile.lock with new wpmreleasetoolkit version
+
+  ### CHANGELOG.md
+  puts '>>> Updating the `CHANGELOG.md` file...'
+  update_changelog_sections(
+    file: 'CHANGELOG.md',
+    wip_header_title: 'Develop',
+    placeholder_sections: ['Breaking Changes', 'New Features', 'Bug Fixes', 'Internal Changes'],
+    new_section_title: new_version
+  )
+
+  puts <<~INSTRUCTIONS
+
+    ---------------
+
+    >>> WHAT'S NEXT
+
+    Please check that the version bump and CHANGELOG.md updates looks ok.
+    Then, commit the changes in a new `release/#{new_version}` branch and create PRs to `develop` and `trunk`.
```

Comment:
Suggestion for the future, should this task create and checkout to the branch, too?

Reply:
Yep, that's part of improvement idea 1 listed in PR description :) I am likely to iterate and improve over that rake task to automate the branch creation, commit, push, and why not even create the PRs.

As for the gem push, we'll probably do it manually at first, but I'm hoping that we'll find some time at some point to make it run by CI (similar to how we do for apps, trigger a CI build via API, which does `gem build` and `gem push`, then create the GitHub PR with the right target for tag to be created once all go green on the push).